### ISSUE TITLE
Bot stopped working 

### ISSUE THREAD

**Issue by duceduc** [futomi/node-switchbot#8]
The bot has been working up until today. I am receiving this error. I have reinstalled the noble and the node-switchbot. Would you know what happened with the following error?

`#devicefound: WoHand (e7:93:41:cc:f8:ea)
#device:press
Error: No service was found.
at /home/pi/node_modules/node-switchbot/lib/switchbot-device.js:228:18
at Peripheral.<anonymous> (/home/pi/node_modules/@abandonware/noble/lib/peripheral.js:81:7)
at Object.onceWrapper (events.js:422:26)
at Peripheral.emit (events.js:315:20)
at Noble.onServicesDiscover (/home/pi/node_modules/@abandonware/noble/lib/noble.js:309:16)
at NobleBindings.emit (events.js:315:20)
at NobleBindings.onServicesDiscovered (/home/pi/node_modules/@abandonware/noble/lib/hci-socket/bindings.js:317:8)
at Gatt.emit (events.js:315:20)
at Object.callback (/home/pi/node_modules/@abandonware/noble/lib/hci-socket/gatt.js:369:12)
at Gatt.onAclStreamData (/home/pi/node_modules/@abandonware/noble/lib/hci-socket/gatt.js:133:26)`

**Update**: A reboot of the server will work for a few minutes, then bot stop responding. These are the errors I see.
error: Error: No service was found.
error: Error: Failed to discover services and characteristics: DISCONNECTED

**Comment by futomi** [comment]
BLE is really unstable. You should catch the exception and try to connect the device over and over again. Though the node-switchbot try to connect device some times, it is possible to fail depending on your environment.

It may be that the battery level of your bot is low. You should check the level.

**Comment by duceduc** [comment]
After a few days of tinkering, I figured out why it stopped working. My **update** in the first post should have given me a clue why.
After I got the bot to work, I created another node project to scan data info every 5 minutes via cronjob with the same bot devices. Apparently, this caused a conflict with the first node project thus giving me the error. I have since removed the second project and the bot is responding as it should.
